packages/core-app-api/src/apis/system/ApiProvider.test.tsx

```diff
@@ -55,7 +55,9 @@ describe('ApiProvider', () => {
         <MyHocConsumer />
       </ApiProvider>,
     );
-    renderedHoc.getByText('hoc message: hello');
+    expect(renderedHoc.getByText('hoc message: hello').innerHTML).toEqual(
```

Comment:
.toBeInTheDocument is our preferred assertion. this is a bit of a duplicate since we're already verifying the text content with the get